doc/sr_cpp_walkthrough.rst

```diff
@@ -3,21 +3,19 @@ C++
 ***
 
 
-In this section, examples are presented using the SmartRedis C++
-API to interact with the RedisAI tensor, model, and script
-data types.  Additionally, an example of utilizing the
-SmartRedis ``DataSet`` API is also provided.
+This section presents examples that use the SmartRedis C++ API to 
+interact with the RedisAI tensor, model, and script data types. 
+Additionally, this section demonstrates an example of utilizing the SmartRedis ``DataSet`` API.
 
 
 
 .. note::
       The C++ API examples rely on the ``SSDB`` environment
-      variable being set to the address and port of the Redis database.
+      variable set to the address and port of the Redis database.
 
 
 .. note::
-    The C++ API examples are written
-    to connect to a clustered database or clustered SmartSim Orchestrator.
+    The C++ API examples connect to a clustered database or clustered SmartSim Orchestrator.
```

Comment:
Not sure I like this change. It's only a minor tweak to make the examples switch to non-clustered (standalone) Redis

Reply:
What about -> "The C++ API examples connect either to a clustered database or clustered SmartSim Orch"

I am trying to avoid the use of 'are'

Reply:
wait thats basically the same as before